tools/setupResourceScripts/datarepo_datasets_outputs.json

```diff
@@ -1 +1 @@
-[{"V2F_GWAS_Summary_Statistics": {"dataset_id": "509ecb17-d442-420e-b9de-2870c858d931", "snapshot_ids": ["4f64f8b0-efa2-49bc-a509-7b7b27cdfe33"]}}]
```

Comment:
We probably shouldn't check in the outputs - would you mind adding it to gitignore?